app/uk/gov/hmrc/cgtpropertydisposalsfrontend/config/ViewConfig.scala

```diff
@@ -179,4 +180,13 @@ class ViewConfig @Inject() (servicesConfig: ServicesConfig) {
 
   val furtherReturnsEnabled: Boolean = servicesConfig.getBoolean("further-returns.enabled")
 
+  logger.info {
+    val flags = List(
+      "periodOfAdminEnabled"     -> periodOfAdminEnabled,
+      "languageSwitchingEnabled" -> enableLanguageSwitching,
+      "furtherReturnsEnabled"    -> furtherReturnsEnabled
+    )
+    s"Application started with feature flags: [${flags.mkString("; ")}"
+  }
```

Comment:
when is this going to be useful? we should be confident in the release process to know what is being shipped - it would be the same effort if not less to look at the config files than trawl through logs to see what has been switched on

Reply:
this is something @rszmacinski wanted while he was testing to see what was enabled or not. It does no harm so I just put it in